jetstream_pt/cache_manager.py

```diff
@@ -654,3 +654,61 @@ def finalize(self):
             self.new_v_scaler,
             self.input_pos,
         )
+
+
+class PageKVCacheGenerate:
+  """Page attention kvache generator without quantization"""
+
+  def __init__(
+      self,
+      cache_k: torch.Tensor,  # previous cache
+      cache_v: torch.Tensor,  # previous cache
+      page_token_indices: jax.Array,  # page and token indices for the cache
```

Comment:
can this also be torch.Tensor?

Reply:
Yes, page_token_indices can also be a torch.Tensor. Updated